cmd/nerdctl/stats.go

```diff
@@ -283,17 +283,6 @@ func statsAction(cmd *cobra.Command, args []string) error {
 		// make sure each container get at least one valid stat data
 		waitFirst.Wait()
 
-		var errs []string
```

Comment:
Why remove this block?

Reply:
useless regarding this https://github.com/containerd/nerdctl/pull/1298/files#diff-8541fa08e70d092bb039071593918dac552c3cbf1088d15e71d0f6117ad4b249R306

the error display should be within the ticker